### PR TITLE
895 various fixes

### PR DESCRIPTION
**Pull request by johangirod**

### CODE REVIEW

File: components-ui/button/index.tsx [hunk 1]
```diff
@@ -1,8 +1,7 @@
 'use client';
```
Comment (XavierJp) [inline components-ui/button/index.tsx:1]: No need to use it as a client component anymore ?

File: utils/server-side-helper/app/error-page.ts
Comment (XavierJp) [inline utils/server-side-helper/app/error-page.ts:1]: Weirdly this appear as an empty file on github

**Review by XavierJp — APPROVED** [review]

File: components-ui/button/index.tsx (hunk 1 above)
Comment (johangirod) [reply components-ui/button/index.tsx:1]: I think we do, as there is an event callback forwarding (onClick). I'll check on that.

File: utils/server-side-helper/app/error-page.ts
Comment (johangirod) [reply utils/server-side-helper/app/error-page.ts:1]: I think I pushed the empty file, my bad.

File: components-ui/button/index.tsx (hunk 1 above)
Comment (johangirod) [reply components-ui/button/index.tsx:1]: Confirmed here: https://react.dev/reference/react/use-client#limitations

**Pull request merged**